### PR TITLE
feat: Update to Emoji 15

### PR DESCRIPTION
**Pull request by susnux**
Update the emoji data to version 15.

### CODE REVIEW

File: docs/app.vue [hunk 1]
```diff
@@ -281,7 +281,7 @@ export default {
       let emoji = index.findEmoji(':smile:')
       // Note, that position in the emoji sheet is calculated by
       // `emoji` object
-      let style = `background-position: ${emoji.getPosition()}; background-image: url(https://unpkg.com/emoji-datasource-twitter@14.0.0/img/twitter/sheets-256/64.png); width: 24px; height: 24px; display: inline-block; background-size: 5700%`
+      let style = `background-position: ${emoji.getPosition()}; background-image: url(https://unpkg.com/emoji-datasource-apple@15.0.0/img/apple/sheets-256/64.png); width: 24px; height: 24px; display: inline-block; background-size: 5700%`
```
Comment (serebrov) [inline docs/app.vue:284]: ❓ Is the `twitter` -> `apple` change functional (something did not work) or just visual (apple looks better?)?

**Review by serebrov — CHANGES_REQUESTED** [review]
Thanks for the PR! Please check my question.

Also, some tests are failing, but that is expected - emojis changed positions and we need to update test snapshots, as described [here](https://github.com/serebrov/emoji-mart-vue#updating-emoji), please run `npm run jest -- -u` to update snapshots.

I changed the base for this PR to a branch instead of `master`, so I can test it more after merge before pushing to `master`.

File: docs/app.vue (hunk 1 above)
Comment (susnux) [reply docs/app.vue:284]: I can revert this, mostly this was from a different change (twitter emojis are discontinued and do not provide Emoji 15).

**Comment by serebrov** [comment]
@susnux Thanks! I merged your change to a branch. I will continue (test/merge/release) in #273.

**Pull request merged**